builder/builder_test.go

```diff
@@ -20,6 +20,7 @@ type BuilderTest struct {
 	Query      string
 	NamedQuery string
 	Args       []interface{}
+	NamedArgs  map[string]interface{}
```

Comment:
I think you could remove this `NamedArgs` since we could infer them using `Args` in our test.